python/cog/predictor.py

```diff
@@ -1,15 +1,16 @@
 from abc import ABC, abstractmethod
-from collections.abc import Generator
+from collections.abc import Iterator
```

Comment:
Why the change from Generator to Iterator?

Reply:
It's a subset of the functionality of `Generator` that is neater to write (`Iterator[str]` instead of `Generator[str, None, None]`).

See here: https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes

`Generator` is a subclass of `Iterator`.